var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -12,30 +36,106 @@ class Adios(Package):
     """
 
     homepage = "http://www.olcf.ornl.gov/center-projects/adios/"
-    url = "https://github.com/ornladios/ADIOS/archive/v1.9.0.tar.gz"
+    url      = "https://github.com/ornladios/ADIOS/archive/v1.10.0.tar.gz"
+
+    # make -j is currently not supported
+    parallel = False
+
+    version('1.10.0', 'eff450a4c0130479417cfd63186957f3')
+    version('1.9.0' , '310ff02388bbaa2b1c1710ee970b5678')
+    version('1.8.0' , 'c9116ec31e6b386f0e6ea5ab675c57e9')
```

Comment:
I know a user with adios 1.3.1 dependency, with his own fix in his adios
copy :-(

On Mon, Aug 29, 2016 at 5:14 PM, Denis Davydov <notifications@github.com>
wrote:

> In var/spack/repos/builtin/packages/adios/package.py
> <https://github.com/LLNL/spack/pull/1652#discussion_r76688738>:
>
> > @@ -12,30 +36,106 @@ class Adios(Package):
> >      """
> >
> >      homepage = "http://www.olcf.ornl.gov/center-projects/adios/"
> > -    url = "https://github.com/ornladios/ADIOS/archive/v1.9.0.tar.gz"
> > +    url      = "https://github.com/ornladios/ADIOS/archive/v1.10.0.tar.gz"
> > +
> > +    # make -j is currently not supported
> > +    parallel = False
> > +
> > +    version('1.10.0', 'eff450a4c0130479417cfd63186957f3')
> > +    version('1.9.0' , '310ff02388bbaa2b1c1710ee970b5678')
> > +    version('1.8.0' , 'c9116ec31e6b386f0e6ea5ab675c57e9')
>
> there has already been the 1.9.0 dependency and I want to avoid breaking
> downstream code.
>
> 👍 If you have not tested 1.8-1.6, you can remove from here until someone
> needs them and tests the package.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LLNL/spack/pull/1652/files/88254a14a7b45efcfbe12496cb6f68fe0d2156ae#r76688738>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADGMLTiZ9KqSSE-Zjnd709qTeW6_ZWdUks5qk0vHgaJpZM4Jvkjc>
> .
>